app/make_mwa_tied_array_beam.c

```diff
@@ -568,6 +574,15 @@ void make_tied_array_beam_parse_cmdline(
                 case 'n':
                     opts->nchunks = atoi(optarg);
                     break;
+                case 'N':
+                    opts->out_nstokes = atoi(optarg);
+                    if ((opts->out_nstokes != 1) && (opts->out_nstokes != 4))
```

Comment:
I think this should be an "OR" rather than "AND"?

Reply:
I started with "OR" and realised I need both of these to be true actually, so "AND" is the right operator.

Reply:
Oh yes. I misread it, but for the case where you want it to FAIL then it's correct. "If it's not 1 and not 4, those are the only allowed options, so fail."